server/text_generation_server/layers/fp8.py

```diff
@@ -92,9 +123,17 @@ def get_weights(self, weights: "Weights", prefix: str):
                 .reshape(-1)
                 .expand(w.shape[0])
             )
+            try:
+                input_scale = weights.get_tensor(
```

Comment:
`Weights` also has `_has_tensor` maybe we should make it public and use it here?

Reply:
Same for `try: [...]get_tensor` below.

Reply:
Updated to use the `has_tensor`

Reply:
Updated to use the `has_tensor`